coderedcms/blocks/__init__.py

```diff
@@ -82,6 +83,10 @@
 ]
 
 CONTENT_STREAMBLOCKS = HTML_STREAMBLOCKS + [
+    ('accordion', AccordionBlock([
```

Comment:
Does the streamfield feel noticeably slower after this change? I know that every time we add another nested block it's an n-squared increase in page load time when rendering the streamfield (at least used to be, not sure in current version of wagtail?)

Reply:
I didn't notice any particular slow down when testing locally.